### ISSUE TITLE
Test build during the week

### ISSUE THREAD

**Issue by Gonza10V** [iGEM-Engineering/igem-actions#6]
the script got 7 out of 10 parts.

**Comment by jguillen15** [comment]
It seems there are 2 main errors:
- In scripts/test/test_import.py::TestImportParts::test_all_import_types, 2 Sequences couldn't be retrieved : 
'http://sevahub.es/public/Canonical/cd_OriT' and 'http://sevahub.es/public/Canonical/lacZ_pUC19', so when the comparison is made for the test(against the comparison file we have) it fails. When I try to go to those links they don't exist anymore, so my guess is that the sequences were deleted and therefore couldn't be retrieved.
- In scripts/test/test_import.py::TestImportParts::test_import, the generated NCBI_GenBank_imports.gb file is not identical to what it is expected to be.

**Comment by jakebeal** [comment]
It looks like SevaHub no longer has any of its public data accessible. I've emailed Angel, but for now let's just take those out of the set of devices that are being tested.

With regards to the NCBI imports, have you checked what the difference is? If it's due to NCBI changing what its contents are, then we should just update the file contents to match the current contents.

**Comment by jguillen15** [comment]
I just checked it, it seems that NCBI changed some metadata: [https://www.ncbi.nlm.nih.gov/nuccore/JWYZ01000115.1](https://www.ncbi.nlm.nih.gov/nuccore/JWYZ01000115.1) (I assumed this is the sequence that is being used in the test)
This is the file we are using: [https://github.com/iGEM-Engineering/iGEM-actions/blob/main/scripts/test/test_files/NCBI_GenBank_imports.gb](https://github.com/iGEM-Engineering/iGEM-actions/blob/main/scripts/test/test_files/NCBI_GenBank_imports.gb)
I changed the file and ran the test again, but it is still failing and created one more issue, with the package.nt collated file.
I will keep investigating further.

**Comment by jakebeal** [comment]
Yes, looks like the metadata has changed, so please set up a pull request updating to the new metadata.
Also, per the message from Angel, the  SevaHub sequences should be back up again.

**Comment by jguillen15** [comment]
Yes, I can see the SevaHub sequences back up again and passing the test!
However, replacing the NCBI_GenBank_imports.gb file with the new metadata didn't solve the issue.
There seems to be a confusion regarding the sequence URL: [https://www.ncbi.nlm.nih.gov/nuccore/JWYZ01000115.1](https://www.ncbi.nlm.nih.gov/nuccore/JWYZ01000115.1)
During the test, a file with the same content is generated in a temporary subdirectory, but it has the URL : [https://www.ncbi.nlm.nih.gov/nuccore/JWYZ01000115_1](https://www.ncbi.nlm.nih.gov/nuccore/JWYZ01000115_1)
This small change in the URL, from "." to "_", redirects to a search page rather than to the Genbank sequence itself.
I have trying to understand why this change is introduced in the test, but I haven't figured it out yet.

**Comment by jakebeal** [comment]
That URL isn't necessarily wrong - an SBOL `displayID` property isn't allowed to have a "." in it, so when the identifier is converted to SBOL, the `JWYZ01000115.1` would, in fact, be likely to change to `JWYZ01000115_1`, which is _not_ expected to be a literal URL for NCBI.

I don't think that's actually where your error actually is, however, since the error you're getting is comparison of the .gb files, and that URL isn't in the .gb file anywhere anyway. You'll need to look at what the actual difference is that's causing the error report.

**Comment by jakebeal** [comment]
Actions are still failing in the develop branch, so I don't think this is done until we get in a PR that fixes this.
